cryptographic_flows.md

```diff
@@ -1,27 +1,24 @@
 
-# Authenticate
-The local client inputs a user ID and a password and receives an "export key", which we will consider as a valid symmetric encryption key, as output. The key server receives as output some user-specific authentication material, which it can compare to its stored data to verify authentication.
-
-Both parties recieve an open channel as output, which satisfies mutual authentication, confidentiality, and integrity.
-
-
 # Generate a secret
 
-The local client component can generate a secret. Using a cryptographically secure pseudo-random number generator with a random seed, generate 256 bits of output.
-
-Generate a unique tag to associate with the secret (unique among secrets stored by the local client).
+The local client component can generate a secret.
+1. Generate the secret: using a cryptographically secure pseudo-random number generator with a random seed, generate 256 bits of output.
```

Comment:
What cryptographic dependency should we rely on for the PRNG? For production of the random seed?